Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Definitions/HolographicFrameNativeData.cs

```diff
@@ -24,12 +24,12 @@ public struct HolographicFrameNativeData
         public uint MaxNumberOfCameras;
 
         /// <summary>
-        /// The current native root <see href="https://docs.microsoft.com/uwp/api/windows.perception.spatial.spatialcoordinatesystem">ISpatialCoordinateSystem</see>).
+        /// The current native root <see href="https://docs.microsoft.com/uwp/api/windows.perception.spatial.spatialcoordinatesystem">ISpatialCoordinateSystem</see>.
```

Comment:
is this the root pointer for all native access? not sure what "root" means here

Reply:
It's the `ISpatialCoordinateSystem` at the root of the scene. Happy to use a different word if something else makes it more clear!

Reply:
ignore me.... i missed the <see ... /> part of the comment. it is fine as is

Reply:
Error:BaseWindowsMixedRealityCameraSettings.cs(54,17): error CS0246: The type or namespace name 'WindowsMixedRealityCameraSettingsProfile' could not be found (are you missing a using directive or an assembly reference?)

Reply:
@LongForever1993 Where and when are you seeing that error? Please file an issue if it's consistent.